juriscraper/opinions/united_states/federal_district/dcd.py

```diff
@@ -4,142 +4,102 @@
+            # reflect this, but they do classify these separately under
+            # these 'mj' and 'mc' codes and the first page of these
+            #  documents will often refer to them as 'Magistrate Case
+            # ####-####' so, we will too.
+            elif nature_code == "mj" or "mc":
+                nature_of_suit = "Magistrate Case"
             else:
-                document_numbers.append(url)
-        return document_numbers
+                nature_of_suit = "Unknown"
 
-    def _get_judges(self):
-        judges = []
-        rowpath = "//table[2]//tr[position()>0]"
-        urlpath = "./td[3]/a/@href"
-        judgepath = "./td[3]"
-        for row in self.html.xpath(rowpath):
-            for judge_element in row.xpath(judgepath):
-                judge_string = html.tostring(
-                    judge_element, method="text", encoding="unicode"
-                )
-                judge = re.search(
-                    r"(by\s)(.*)", judge_string, re.MULTILINE
-                ).group(2)
-                # Determine the number of urls in each row and pad the judges
-                # list sufficiently
-                count = len(row.xpath(urlpath))
-                judges.extend([judge] * count)
-        return judges
+        return doc_number, nature_of_suit
+
+    def _get_docket_document_numbers(self):
+        return [case["docket_document_numbers"] for case in self.cases]
 
     def _get_nature_of_suit(self):
-        nos = []
-        for url in self.html.xpath(
-            "//table[2]//tr[position()>0]/td[3]/a/@href"
-        ):
-            # Because we are acting directly on the entire url list, no padding
-            # of the nature of suit field is required.
-            regex = r"(\?)(\d+)([a-z]+)(\d+)(\-)(.*)"
-            url_str = re.search(r"(\?)(\d+)([a-z]+)(\d+)(-)(.*)", url)
-            # In 2012 (and perhaps elsewhere) they have a few weird urls.
-            if re.search(regex, url) is not None:
-                nature_code = url_str.group(3)
-                if nature_code == "cv":
-                    nos.append("Civil")
-                elif nature_code == "cr":
-                    nos.append("Criminal")
-                # This is a tough call. Magistrate Cases are typically also
-                # Criminal or Civil cases, and their docket_number field will
-                # reflect this, but they do classify these separately under
-                # these 'mj' and 'mc' codes and the first page of these
-                #  documents will often refer to them as 'Magistrate Case
-                # ####-####' so, we will too.
-                elif nature_code == "mj" or "mc":
-                    nos.append("Magistrate Case")
-                else:
-                    nos.append("Unknown")
-            else:
-                nos.append("Unknown")
-        return nos
+        return [case["nature_of_suit"] for case in self.cases]
```

Comment:
is this necessary?

Reply:
I think it is, since OpinionSiteLinear doesn't have a getter for that field, and AbstractSite has an empty getter
If you run a test scraper, `site.cases[0]` will have the nature_of_suit data, but `site[0]` (which is used by CL) will not

Reply:
I thin the solution is to add that field to opinionsitelinear